spec/factories/chats.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :chat do
+    match
```

Comment:
Maybe you are already using Faker, if not it's a good moment to get in touch with that gem